### DISCUSSION TITLE
Resume Bundle x ATS Scoring

### DISCUSSION THREAD

**Discussion by AmruthaPariprolu** [recodehive/resume-pitch#88] [Q&A]
Resume Link = 
[AmrFinal.pdf](https://github.com/user-attachments/files/17005634/AmrFinal.pdf)
LinkedIn Profile = https://www.linkedin.com/in/amrutha-pariprolu-b812892a6/
github progile = https://github.com/AmruthaPariprolu

I am a 3rd year undergrad student pursuing CSE(AIML) at JNTUH.

**Comment by sanjay-kv** [comment]
I didnt see any topmate request for this 
could you check that again: 
if you wanna book again: https://topmate.io/sanjaykv/1047284/pay
use the same freecode you used before.
